themes/inbeat/assets/scss/_base.scss

```diff
@@ -5,6 +5,10 @@ body {
     margin: 0;
 }
 
+[v-cloak] {
+  display: none;
+}
```

Comment:
What's the goal here?

Reply:
Used to hide content until compilation is finished! Prevent popup glitching when loading page

Reply:
Damn, didn't know about that feature! Sorry